android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java

```diff
@@ -443,6 +443,16 @@ public static void makeFieldHidden(JSONObject fieldObject) {
         }
     }
 
+    public static void removeDeletedInvalidFields(String fieldKeyPrefix, Map<String, ValidationStatus> invalidFields, ArrayList<String> removeThisFields) {
+        for (String removeField : removeThisFields) {
```

Comment:
Prefer nouns to verbs for variables e.g. `fieldToRemove` from `fieldsToRemove` as opposed to `removeField` and `removeTheseFields.`